### PR TITLE
Added getPlaybackRate() method

### PR DESCRIPTION
**Pull request by Kage1989**
Unable to use this.setPlaybackRate(1) in this.getPlaybackRate(). 

An "Uncaught TypeError: Cannot read property '0' of undefined" error would occur at isPaused() method.

### CODE REVIEW

File: src/mediaelement.js [hunk 1]
```diff
@@ -86,6 +86,16 @@ WaveSurfer.util.extend(WaveSurfer.MediaElement, {
...
     },
+    
+    /**
+     * Get the audio source playback rate.
+     */
+    getPlaybackRate: function () {
+    	if (this.media.playbackRate == null) {
+    		this.media.playbackRate = 1;
+    	}
+        return this.media.playbackRate;
```
Comment (katspaugh) [inline src/mediaelement.js]: I'd rather not set a value in the getter. How about just `return this.media.playbackRate || 1`?

File: src/mediaelement.js [hunk 2]
```diff
@@ -86,6 +86,16 @@ WaveSurfer.util.extend(WaveSurfer.MediaElement, {
         this.playbackRate = value || 1;
         this.media.playbackRate = this.playbackRate;
     },
+    
+    /**
+     * Get the audio source playback rate.
+     */
+    getPlaybackRate: function () {
```
Comment (katspaugh) [inline src/mediaelement.js]: Actually, there's no need for this method, as it's inherited from the Web Audio backend.

**Comment by Kage1989** [comment]
Ok. I'll change accordingly.

**Comment by katspaugh** [comment]
Thanks! By the way, you don't need to create new PRs, you can just commit and push to the same branch of yours.

**Comment by Kage1989** [comment]
oh. i didn't know that. ok. last pr from me regarding this issue. closing the original request as well.

**Pull request closed without merging**